sdk/mixedreality/azure-mixedreality-authentication/src/test/java/com/azure/mixedreality/authentication/MixedRealityStsClientTestBase.java

```diff
@@ -40,13 +43,21 @@ HttpPipeline getHttpPipeline(HttpClient httpClient) {
         final List<HttpPipelinePolicy> policies = new ArrayList<>();
         policies.add(new BearerTokenAuthenticationPolicy(credential, authenticationScope));
 
-        if (getTestMode() == TestMode.RECORD) {
+        if (interceptorManager.isRecordMode()) {
```

Comment:
sanitizers are to be added in both playback and record mode to match correctly with redacted content.

Reply:
I assume that's applicable only if we're redacting RequestUri and Headers ? as those are what's used to match the request ?
Else, I should see failures in CI.

Reply:
We pull up the match using the request URI but after that we just match everything, I see you have added BodilessMatcher it coul be why we don't see the match errors.